nixos/modules/system/boot/binfmt.nix

```
@@ -280,28 +278,50 @@ in {
         '';
         type = types.listOf (types.enum (builtins.attrNames magics));
       };
+
+      preferStaticEmulators = mkOption {
+        default = "false";
```

```suggestion
        default = false;
```
Shouldn't this be a bool?